styles.css

```diff
@@ -0,0 +1,458 @@
+@import url('https://fonts.googleapis.com/css2?family=Open+Sans:ital,wght@0,300..800;1,300..800&display=swap');
+@import url('https://fonts.cdnfonts.com/css/metropolis-2');
+
+* {
+    margin: 0;
+    padding: 0;
+    font-family: 'Metropolis', sans-serif;
+    color: #000;
+}
+
+header {
```

Comment:
Please avoid using tag selectors. Someone other will be surprised when they will try to use the basic tag and the styling will be different.